sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/LogsHelper.cs

```diff
@@ -56,17 +56,27 @@ internal static List<TelemetryItem> OtelToAzureMonitorLogs(Batch<LogRecord> batc
             {
                 try
                 {
-                    telemetryItem = new TelemetryItem(logRecord, resource, instrumentationKey);
                     if (logRecord.Exception != null)
                     {
+                        telemetryItem = new TelemetryItem("Exception", logRecord, resource, instrumentationKey);
                         telemetryItem.Data = new MonitorBase
                         {
                             BaseType = "ExceptionData",
                             BaseData = new TelemetryExceptionData(Version, logRecord),
                         };
                     }
+                    else if (logRecord.CategoryName == "Azure.Monitor.OpenTelemetry.CustomEvents")
```

Comment:
can we use referenceequals as we use this string within the same library?

Reply:
My understanding is it uses that by default.